inspectit.shared.all/src/main/java/rocks/inspectit/shared/all/storage/nio/ByteBufferProvider.java

```diff
@@ -252,11 +252,13 @@ protected void updatePoolProperties() {
 		// assume that the maxDirect memory is 64MB
 		long maxDirectMemory = 64 * 1024 * 1024;
 		try {
-			if (UnderlyingSystemInfo.JVM_PROVIDER.equals(JvmProvider.SUN) || UnderlyingSystemInfo.JVM_PROVIDER.equals(JvmProvider.ORACLE)) {
+			if (UnderlyingSystemInfo.JVM_PROVIDER.equals(JvmProvider.SUN) || UnderlyingSystemInfo.JVM_PROVIDER.equals(JvmProvider.ORACLE) || UnderlyingSystemInfo.JVM_PROVIDER.equals(JvmProvider.AZUL)) {
```

Comment:
Can you remove this JVM check and try to get the 'maxDirectMemory' value from the JVM each time and fall back to 64MB if it does not work? Then, we don't have to maintain this list and are not missing any JVM which provides the value.